### PR TITLE
feat: add registration panel

### PR DESCRIPTION
**Pull request by Valentine0604**
Hello friend, please review my code, thank you from the mountain :D

### CODE REVIEW

File: src/Registration.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+public class Registration {
+    private String nickname;
+    private String password;
+    private String email;
+    private String youCatFavouriteToy;
```
Comment (N33Qu) [inline src/Registration.java]: Pls delete this variable and its getter and setter

File: src/Registration.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
+public class Registration {
+    private String nickname;
```
Comment (N33Qu) [inline src/Registration.java]: change  variable name to username pls

File: src/Registration.java (hunk 1 above)
Comment (Valentine0604) [reply src/Registration.java]: r u sure, mr?

Comment (N33Qu) [reply src/Registration.java]: yes pls delete and add sth new

Comment (Valentine0604) [reply src/Registration.java]: ok, i will add more cats

File: src/Registration.java (hunk 2 above)
Comment (Valentine0604) [reply src/Registration.java]: i will add more cats

File: src/Registration.java (hunk 1 above)
Comment (N33Qu) [reply src/Registration.java]: whatever i'll let it pass for now

File: src/Registration.java (hunk 2 above)
Comment (N33Qu) [reply src/Registration.java]: ok

**Review by N33Qu — APPROVED** [review]

**Pull request merged**